src/main/kotlin/deadline/application/DeadlineChanger.kt

```diff
@@ -0,0 +1,34 @@
+package deadline.application
+
+import deadline.domain.Deadline
+import deadline.domain.DeadlineRepository
+import deadline.domain.Time
+import deadline.domain.exceptions.AlreadyConfiguredDeadline
+import deadline.domain.exceptions.NonExistentDeadlineException
+import shared.domain.Identifier
+import shared.domain.exceptions.InvalidUUIDException
+
+class DeadlineChanger(private val repository: DeadlineRepository) {
+
+    /***
```

Comment:
Should throw `AlreadyUsedIdentifierException` if the identifier provided in the new primitives is already used in the repository

Reply:
True